panels/history/ha-panel-history.html

```diff
@@ -114,9 +132,12 @@
 
   attached: function () {
     this.datePicker = new window.Pikaday({
-      field: this.$.datePicker.inputElement,
+      field: document.createElement('input'),
+      trigger: this.$.datePicker.inputElement,
```

Comment:
I created a dummy element for the field, and used the actual element as the pikaday trigger.

When clicking on the date that was already selected, the pikaday updated the field value using its internal formatting, and the nuclear binding was never triggered to rewrite it since the same day was selected. This change allows us to capture the date select event from the pikaday without it actually rewriting the field value.